Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM registry.ci.openshift.org/openshift/release:golang-1.14 AS builder
```

Comment:
can you change to docker.io/amd64/golang:1.14 so we will not be dependent on openshift ci access?